cmd/kube-apiserver/app/server.go

```diff
@@ -68,6 +68,7 @@ type APIServer struct {
 	BindAddress                net.IP
 	AdvertiseAddress           net.IP
 	SecurePort                 int
+	ReadWritePort              int
```

Comment:
Though I would (simultaneously) question any installation that disabled TLS (even self signed). Is the issue that credentials are required to connect to the TLS endpoint? Would an "allow anonymous" authenticator help bridge this use case?